sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/Customized/Models/UpdateModelSettingsOptions.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.Iot.TimeSeriesInsights.Customized
```

Comment:
namespace should match the namespace of the client code. So Azure.Iot.TimeSeriesInsights